tiledb/sm/config/config.h

```diff
@@ -207,6 +207,24 @@ class Config {
    */
   static const std::string SM_MEM_SPARSE_UNORDERED_WITH_DUPS_RATIO_ARRAY_DATA;
 
+  /**
+   * The maximum count of partitions to be generated via CAPI subarray
+   * partitioning.
+   */
+  static const std::string SP_SUBARRAY_PARTITIONER_RESULT_COUNT;
+  /**
+   * TBD: explain me
```

Comment:
Are you trying to create a setting for this here? I can show you, schedule some time with me.